### DISCUSSION TITLE
Declare input and output files in configuration

### DISCUSSION THREAD

**Discussion by will-stone** [tailwindlabs/tailwindcss#5033] [Ideas]
Hi 👋 

Not sure if this would have any downsides, but how about declaring the input/output files within the configuration file, in order to make the CLI cleaner? e.g.

```js
// tailwind.config.js

module.exports = {
  input: './src/tailwind.css',
  output: './dist/tailwind.css',
}
```

```bash
npx tailwindcss
# or
npx tailwindcss --watch
```

Keep up the great work. Happy TW user over here 😄

**Comment by simonswiss** [comment]
I think this is a nice idea!

That reminds me of how [eleventy](https://www.11ty.dev/docs/config/) handles it, and I quite like it:

![image](https://user-images.githubusercontent.com/485747/126599376-4a545f66-036e-4d37-b3ad-0f4649fc24d5.png)

**Comment by changethe** [comment]
this would be really neat, came here to ask the same

**Comment by Montago** [comment]
Has someone made this work yet ?

**Comment by gerardreches** [comment]
I'm just new to Tailwind and this is one of the first things I would like to do.

**Comment by mybearworld** [comment]
Multiple CSS files can use the same config.

**Reply by Montago** [reply]
For at single project, you only have 1 config file... 

what do you mean with multiple tailwind files ?

**Reply by mybearworld** [reply]
Not necessarily, sometimes, you need multiple config and CSS files.

**Reply by Montago** [reply]
well, for all of us, who only have 1 input css file and 1 config file - it would be amazing if it was defined in the tailwind file instead of using input parameters in the builder every time. 

Luckily i've found a way around this issue by integrating the Tailwind build step into the build config - HOWEVER - this doesn't solve the case where i want to have the watcher running with auto-build :-/
